.pre-commit-config.yaml

```diff
@@ -169,6 +169,12 @@ repos:
             pandas/tests/io/excel/test_writers\.py|
             pandas/tests/io/pytables/common\.py|
             pandas/tests/io/pytables/test_store\.py$
+    -   id: no-pd-api-types
+        name: Check code for instances of pd.api.types
+        entry: pd\.api\.types
```

Comment:
can you make this `(pd|pandas)` (not sure if this is a full regex or not

Reply:
:+1:

not sure what you mean by "full regex", but this is just `re.search`:

https://github.com/pre-commit/pre-commit/blob/ebd83171dfa8030a9767018b231b6fb1831b082b/pre_commit/languages/pygrep.py#L25-L26

Reply:
Yes, I guess I need to add it and also change the ```types``` to ```types.``` , so it won't catch the imports but only the usages in the code itself...

Example:

Good:
```from pandas.pai.types import is_number```

Not Good:
```pandas.api.types.is_number(1)```

Reply:
right